src/main/java/module-info.yml

```diff
@@ -0,0 +1,10 @@
+name: com.zaxxer.nuprocess
+requires:
+  - module: com.sun.jna
+exports:
+  - package: com.zaxxer.nuprocess
+  - package: com.zaxxer.nuprocess.codec
+  - package: com.zaxxer.nuprocess.internal
```

Comment:
I think `internal`, `linux`, `osx` and `windows` should likely not be exported. They're all intended to be implementation details of using the API defined in `com.zaxxer.nuprocess`. For whatever use case(s) you're trying to support, is there a reason you're using those packages?